tox.ini

```diff
@@ -7,75 +7,59 @@ requires = pip
 [testenv:py36]
 basepython=python3.6
 
-[testenv:py37]
-basepython=python3.7
-
 [testenv:py38]
 basepython=python3.8
 
 [testenv]
 deps=
-    pytest
-    pytest-cover
-    pytest-randomly
-    pytest-xdist
-    pytest-sugar
-    pytest-instafail
-    pytest-asyncio
-    pytest-postgresql
-    pytest-env
-    pytest-timeout
-    psutil
-    psycopg2
-    openapi-spec-validator
-    more-itertools
+    -rrequirements.dev.txt
+    -rrequirements.txt
 extras=
     dataflow_graphic
-install_command=pip install -c requirements.txt {opts} {packages}
 commands=py.test --log-level DEBUG --cov=inmanta --cov-report term --cov-report xml --junitxml=junit-{envname}.xml -vvv tests/
 # The HOME environment variable is required for Git to discover the user.email and
 # user.name config options (required by test case: tests/test_app.py::test_init_project)
 passenv=SSH_AUTH_SOCK ASYNC_TEST_TIMEOUT HOME
 
 [testenv:pep8]
 deps=
-    flake8
-    pep8-naming
-    flake8-black
-    flake8-isort
-    flake8-copyright
+    -rrequirements.dev.txt
 commands = flake8 src tests tests_common
 basepython = python3
 
 [testenv:isort]
 deps=
-    isort
+    -rrequirements.dev.txt
 commands =
     isort --verbose --check -sp setup.cfg --diff -rc src tests tests_common
+install_command=pip install -c requirements.dev.txt -c requirements.txt {opts} {packages}
```

Comment:
does `-c` still allow `inmanta-dev-dependencies[pytest,async,extension]==1.13.0` with the new pip?

Reply:
I got mixed results here. So that is why I thought I removed it everywhere. It works on my machine, it works for the extensions but not on jenkins for core.

Reply:
Updated. This way I think I finally got to the minimal tox config that does not have to change when adding deps.